care/facility/tests/test_patient_api.py

```diff
@@ -73,7 +73,7 @@ def _get_medical_history_representation(self, history):
 
     def _get_disease_state_representation(self, disease_state):
         if isinstance(disease_state, int):
-            return DiseaseStatusEnum(disease_state).name
+            return DISEASE_STATUS_VALUES.choices(disease_state).name
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [DISEASE_STATUS_VALUES.choices is not callable](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5316739)

Reply:
Since all the tests have passed, this shouldn't be an issue. Also, `choices` is callable here.